lib/mof/scanner.rb

```diff
@@ -144,10 +144,10 @@ def fill_queue
 	when "sint64" then @q.push [:DT_SINT64, CIM::Type.new(m)]
 	when "sint8" then @q.push [:DT_SINT8, CIM::Type.new(m)]
```

Comment:
Might as well allow all the other types references here to be prefixed with `system` as well - at the very least, I think you want to also allow `"system.sint*"`... but I suspect anything in https://msdn.microsoft.com/en-us/library/windows/desktop/aa392392(v=vs.85).aspx would work.